management_interface/management_interface/models.py

```diff
@@ -1,56 +1,92 @@
 import hashlib
+import uuid
 
+from django.contrib.auth.models import User
+from django.core.validators import EmailValidator
 from django.db import models
-import uuid
-from datetime import datetime
+from django.utils import timezone
+
+
+class SecureEmailValidator(EmailValidator):
+    message = "Enter an nhs.net email address"
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs, allowlist=["nhs.net", "thepsc.co.uk"])
```

Comment:
This'll need pulling out.
```suggestion
        super().__init__(**kwargs, allowlist=["nhs.net"])
```

If we want to be able to send test emails either they can come to an Innovation Lab DL or we need to make this configurable per environment.

Reply:
Ok, delete this for a moment.